build.sh

```diff
@@ -95,19 +95,13 @@ trap 'exit_error ${LINENO}' INT ERR
 
 function check_os_release() {
   os_release="${ROOTFS}/etc/os-release"
-  # This shouldn't be required anymore - we pack the rootfs tarball at base level
```

Comment:
Why removing this?

Reply:
This was a straggler from the old build system, this information was added twice.
`VOLUMIO_HARDWARE` and `VOLUMIO_VERSION` are now only after the [rootfs is created](https://github.com/volumio/volumio3-os/blob/be48af7083d42c9d6cb4a530abb4e817e992a788/build.sh#L461).